examples/Python/PET/acquisition_data.py

```diff
@@ -86,13 +86,14 @@ def main():
         acq_data.show(range(dim[1]//4))
     acq_array = acq_data.as_array()
 
-    nv = dim[2]//2
-    views = numpy.arange(nv)
-    acq_subset = acq_data.get_subset(views)
-    dim_subset = acq_subset.dimensions()
-    print('subset dimensions: %d x %d x %d x %d' % dim_subset)
-    if show_plot:
-        acq_subset.show(range(dim[1]//4), title='Sinograms of a subset of views')
+    if storage[0] == 'm':
```

Comment:
Maybe not obvious what this means for a casual reader looking for examples. Would it be better to try/catch and print an explanation?

```suggestion
    if storage[0] == 'm':  # for now, we can only subset acquisition data stored in memory
```